src/v/cluster/health_monitor_backend.cc

```diff
@@ -624,6 +624,7 @@ struct ntp_leader {
     model::term_id term;
     std::optional<model::node_id> leader_id;
     model::revision_id revision_id;
+    size_t size_bytes;
```

Comment:
Can we not make this a part of `ntp_leader` ? Ntp leader should only contain information about leaders

Reply:
Maybe we should introduce `ntp_report` or similar struct that would aggregate both the `ntp_leader` and partition size